device_samples/getting_started/src/send_messages/send_messages_to_central.js

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// To see the dtdl of for this https://github.com/Azure/opendigitaltwins-dtdl
+
+'use strict';
+
+const Protocol = require('azure-iot-device-mqtt').Mqtt;
+const ProvProtocol = require('azure-iot-provisioning-device-mqtt').Mqtt;
+const Client = require('azure-iot-device').Client;
+const Message = require('azure-iot-device').Message;
+const SymmetricKeySecurityClient = require('azure-iot-security-symmetric-key').SymmetricKeySecurityClient;
+const ProvisioningDeviceClient = require('azure-iot-provisioning-device').ProvisioningDeviceClient;
+
+// DPS connection information
+const provisioningHost = process.env.IOTHUB_DEVICE_DPS_ENDPOINT || 'global.azure-devices-provisioning.net';
+const idScope = process.env.IOTHUB_DEVICE_DPS_ID_SCOPE || '';
+const deviceId = process.env.IOTHUB_DEVICE_DPS_DEVICE_ID || '';
+const deviceKey = process.env.IOTHUB_DEVICE_DPS_DEVICE_KEY || '';
+
+const modelIdObject = { modelId: 'dtmi:com:example:Thermostat;1' };
+const telemetrySendInterval = 10000;
+let deviceConnectionString = '';
+
+// check to make sure all env variables are set
+if (provisioningHost === '' || provisioningHost === undefined) {
+  console.error('\x1b[31m%s\x1b[0m', 'Missing Provisioning Host string');
+  process.exit(0);
+}  
+
+if (idScope === '' || idScope === undefined) {
+  console.error('\x1b[31m%s\x1b[0m', 'Missing Id Scope string');
+  process.exit(0);
+}
+
+if (deviceId === '' || deviceId === undefined) {
+  console.error('\x1b[31m%s\x1b[0m', 'Missing Device Id string');
+  process.exit(0);
+}
+
+if (deviceKey === '' || deviceKey === undefined) {
+  console.error('\x1b[31m%s\x1b[0m', 'Missing Device Key string');
+  process.exit(0);
+}  
```

Comment:
nit: this is 20 lines of code that doesn't add any value to the samples.  It could be replaced with 4 asserts

Reply:
@BertKleewein do you have an example on how we could do that?